v3/plugins/redhat/mta/0.0.38/meta.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v2
+version: 0.0.38
+type: VS Code extension
+name: mta
+displayName: Migration Toolkit for Applications (MTA)
+title: Migration Toolkit for Applications (MTA)
+description: Migration Toolkit for Applications (MTA)
+icon: https://raw.githubusercontent.com/windup/rhamt-vscode-extension/master/resources/mta.png
+publisher: redhat
+repository: https://github.com/windup/rhamt-vscode-extension
+category: Other
+firstPublicationDate: "2020-06-25"
+spec:
+  endpoints:
+  - name: "configuration-endpoint"
+    public: true
+    targetPort: 61436
+    attributes:
+      protocol: http
+  - name: "report-endpoint"
+    public: true
+    targetPort: 61435
+    attributes:
+      protocol: http
+  containers:
+  - image: "docker.io/windup3/rhamt-vscode-extension:java8"
+    name: mta-extension
+    memoryLimit: 1500Mi
+    mountSources: true
+    ports:
+      - exposedPort: 61435
+      - exposedPort: 61436
+    volumes:
+      - name: mta-vscode
+        mountPath: "/home/theia/home/theia/redhat.rhamt-vscode-extension"
+  extensions:
+    - https://download.jboss.org/jbosstools/adapters/stable/rhamt-vscode-extension/rhamt-vscode-extension-0.0.37-589.vsix
```

Comment:
The extension version here is different than `0.0.38` -- is this by design or was it an oversight?